code/modules/events/dust.dm

```diff
@@ -64,30 +65,33 @@
 	src.x = startx
 	src.y = starty
 	src.z = level_name_to_num(MAIN_STATION)
-	spawn(0)
-		walk_towards(src, goal, 1)
```

Comment:
Not needed to call this async. The references of DM state it returns instantly and will continue on the background